thirdeye/thirdeye-frontend/app/pods/components/rootcause-select-comparison-range/component.js

```diff
@@ -48,23 +50,15 @@ export default Component.extend({
   compareModeOptions: [
     {
       groupName: 'Weekly',
-      options: [ 'wo1w', 'wo2w', 'wo3w', 'wo4w', 'mean4w', 'median4w', 'min4w', 'max4w' ]
-    },
-    {
-      groupName: 'Hourly',
-      options: [ 'ho1h', 'ho2h', 'ho3h', 'ho6h', 'mean6h', 'median6h', 'min6h', 'max6h' ]
-    },
-    {
-      groupName: 'Daily',
-      options: [ 'do1d', 'do2d', 'do3d', 'do4d', 'mean4d', 'median4d', 'min4d', 'max4d' ]
-    },
-    {
-      groupName: 'Monthly',
-      options: [ 'mo1m', 'mo2m', 'mo3m', 'mo6m', 'mean6m', 'median6m', 'min6m', 'max6m' ]
+      options: [ 'wo1w', 'wo2w', 'wo3w', 'mean4w', 'median4w' ]
     },
     {
       groupName: 'Algorithm',
       options: [ 'predicted', 'none' ]
```

Comment:
Any clue on why we have a `none` type under Algorithm? shall we remove that.

Reply:
Sure I will push that change.